Makefile

```diff
@@ -101,6 +101,9 @@ webapp: ## Build webapp.
 watch-webapp: ## Run webapp watching for changes.
 	cd webapp; npm run watchdev
 
+watch-plugin: ## Run and upload the plugin to a development server
```

Comment:
Can we check that modd is installed, and install it?  Or at least a note here saying how to do it?

Reply:
Sure! Fixed in fd57d81